src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -744,6 +744,25 @@ static AppleBundle createAppleBundle(
         frameworksBuilder.add(graphBuilder.requireRule(dep).getSourcePathToOutput());
       }
     }
+
+    // Albeit above comments, we (Airbnb) don't have any generated framework.
+    // We are simply copying all transitive prebuilt framework dependencies to the bundle.
```

Comment:
Should we consider simply removing the above section instead?

Reply:
We still need above code to copy to direct dependent frameworks if there is any. I also think it's better to keep the change minimum.